packages/dm-core-plugins/src/job/JobControlButton.tsx

```diff
@@ -13,57 +13,70 @@ export const JobControlButton = (props: {
 }) => {
   const { jobStatus, createJob, asCronJob, disabled, exists, remove } = props
   const [hovering, setHovering] = useState(false)
+  const [buttonColor, setButtonColor] = useState<
+    'primary' | 'secondary' | 'danger' | undefined
```

Comment:
Why would the color ever be undefined? 🤔

Reply:
Does not really make sense, removed it 👍